docs/build/html/_sources/gettingstarted.rst.txt

```diff
@@ -0,0 +1,84 @@
+Getting Started
```

Comment:
Why is this here and not in https://github.com/Gepardec/Hogarama.wiki.git ?

Reply:
through this approach, we can version control our documentation the same way we do our code. in addition, we can have snapshot our documentation for multiple releases. e.g. the documentation checked in to release x fits for release x since those are bundled together. you can easily jump from one version to the other and get the corresponding documentation, even if the doc content changes completely in a year from now

Reply:
in addition, we can utilize GitHub pages to serve the docs at gepardec.github.io which we can redirect to a gepardec.com subdomain if we wish to do so

Reply:
*available via the above-mentioned link only once it is merged to master and github pages are enabled for the repository. an example of how this would look like can be locally inspected  by opening docs/build/html/index.html in a browser of your choice

Reply:
There are always more than one way to do it, but if everyone use his own nobody finds anything anymore. Such decisions should be discussed and decided within the project.